docs/pages/repo/docs/ci.mdx

```diff
@@ -5,10 +5,24 @@ description: Recipes for using Turborepo with GitHub Actions, CircleCI, and othe
 
 # Continuous Integration
 
-Turborepo not only speeds up builds, but also your CI pipeline. Below are a few ways to use Turborepo with various Continuous Integration providers.
+Turborepo not only speeds up builds, but also the rest of your CI pipeline by using [Remote Caching](/repo/docs/core-concepts/remote-caching). Below are a few platform recipes to use Turborepo with your Continuous Integration providers.
```

Comment:
Use the full name first (Continuous Integration), and _then_ the acronym (CI) if you're going to use both.